analyze/corruption_triage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
Perhaps it would be better for the analyze directory to live under tools/analyze ?